docs/learn-papers.md

```diff
@@ -0,0 +1,29 @@
+---
+id: learn-papers
+title: Polkadot Papers
```

Comment:
I was unsure of what to actually call this page, if anyone has any ideas, I would greatly appreciate them!